src/utils/notificationUtils.js

```diff
@@ -7,23 +9,37 @@ export const getNotificationProps = notification => {
   const user2Name =
     notification?.message_values?.user2_name || 'Unnamed User';
 
-  // TODO flesh out more once this is included in notifications DEX-739
-  const individual1AdoptionName = deriveIndividualName(
-    notification,
-    'AdoptionName',
-    'Unnamed individual',
+  const yourIndividualName = 'one of the individuals';
+  // notification?.message_values?.user2_name || 'Unnamed Individual'; //TODO update after DEX-927 is resolved
+  const theirIndName =
+    notification?.message_values?.target_individual_name ||
+    'Unnamed Individual';
+  const theirIndividualGuid =
+    notification?.message_values?.target_individual_guid;
+  const theirIndividualName = (
+    <span>
+      <Link
+        newTab
+        external
+        href={`/individuals/${theirIndividualGuid}`}
```

Comment:
This is not a blocking issue, but we've handled this same functionally both as an internal link and an external link. If external is right, then we can remove the changes made to `Link` in #293 
https://github.com/WildMeOrg/codex-frontend/blob/85ed9f2bff315de8ccb5bc6acf332ce6813357aa/src/pages/sighting/encounters/Encounters.jsx#L264-L268

Reply:
I'm pretty sure external was a mistake. I for some reason thought that was required for newTab, but did not check. I will confirm that this was a mistake tomorrow. Good catch! Thanks!

Reply:
not required for newtab!

Reply:
Done